### PR TITLE
Revert runner change for main robustness nightly

### PR DESCRIPTION
**Pull request by jmhbnz**
Until we can resolve issues with fuse for new actuated runner.

Refer: https://github.com/etcd-io/etcd/pull/16824

### CODE REVIEW

File: .github/workflows/robustness-nightly.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
       count: 80
       testTimeout: 200m
       artifactName: main-arm64
-      runs-on: actuated-arm64-8cpu-32gb
+      runs-on: "['self-hosted', 'Linux', 'ARM64']"
```
Comment (serathius) [inline .github/workflows/robustness-nightly.yaml:30]: What about line 46?

Comment (jmhbnz) [reply .github/workflows/robustness-nightly.yaml:30]: Lazyfs tests only run on `main` currently.

Comment (serathius) [reply .github/workflows/robustness-nightly.yaml:30]: Hmm, I think having machine consistent between test scenarios would make it clearer when debugging.

**Review by serathius — APPROVED** [review]

**Review by ahrtr — APPROVED** [review]
LGTM

thx @jmhbnz

**Pull request merged**